apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx

```diff
@@ -94,27 +99,26 @@ export default function useGlobalAnnouncementPopover({
       if (dismissVariant === 'session' && dismissedOnSession) return;
       if (dismissVariant === 'global' && isGlobalAnnouncementExperienced) return;
 
-      // TEMPORARY: only display the white rhino launch popover on the homepage
-      if (asPath !== '/') {
-        return;
-      }
-
       if (authRequired && !isAuthenticated) return;
 
       if (shouldHidePopoverOnCurrentPath) return;
 
       // prevent font flicker on popover load
-      await handlePreloadFonts();
+      // await handlePreloadFonts();
```

Comment:
not sure what the issue is here, but this await never resolved which prevented the code below from executing meaning the modal never appeared. so I commented it out for now